docs/guides/integrate-with-ory-cloud-through-webhooks.mdx

```diff
@@ -32,7 +32,7 @@ welcome email message to all newly registered users.
 
 2. Create a request body Jsonnet snippet and encode it with Base64:
 
-   ```jsonnet title="Mailchimp request body"
+   ```json title="Mailchimp request body"
```

Comment:
@tomekpapiernik jsonnet was correct here

Reply:
https://jsonnet.org

Reply:
oh yea, I know. I changed to `json` to give it syntax highlighting ;) `jsonnet` isn't supported by prism which handles syntax highlighting in docusaurus

Reply:
https://prismjs.com/index.html#supported-languages

Reply:
`json` is a completely different langauge though and not correct. It's like replacing javascript with java.

Once prism adds jsonnet, it will automatically be highlighted. We can also write our own highlighter for prismjs for jsonnet if required. It's also possible that someone else has done this already, and we just import that

Reply:
Sure, you're right. I don't see any `jsonnet` plugins/extenstions for prismjs though :( The main reason why I did that was to enable inline comment I used in this snippet (before it looked the same as the rest of the code).
Also, our docs in the past used `json5` / `json` / `jsonnet` for jsonnet code blocks.

Not sure what to do with that. Social sign-in jsonnet snippets are all `json` and they use a lot of inline comments that will look messy w/o syntax highlighting.

Reply:
It will trip code-formatting though, because e.g. prettier will think that it's JSON. Please fix the metadata to be correct, and we will be able to fix the rest at some point as well! Using JSON as a substitute is a workaround that will cause issues and make it much harder to introduce a jsonnet highlighter later on, because we will need to find all the code samples where we used the incorrect metadata, meaning that we will need to go through every document to fix it!

Reply:
We can probably also set "jsonnet" as an alias for "json", so that it gets the interim syntax highlighting but still is labeled correctly

https://github.com/ory/docs/issues/970